src/common/io.py

```diff
@@ -1546,7 +1546,7 @@ def _get_variable_data_as_trajectory(self,
                 name == f'{DIAGNOSTICS_PREFIX}cpu_time'
             ):
             return Trajectory(
-                time, np.cumsum(self.get_variable_data(f'{DIAGNOSTICS_PREFIX}cpu_time_per_step').x[start_index:stop_index]))
+                time, np.cumsum(self.get_variable_data(f'{DIAGNOSTICS_PREFIX}cpu_time_per_step').x)[start_index:stop_index])
```

Comment:
This means we always get the full trajectory for this particular variable. 
Might not be the most efficient, but is robust; I fear anything else might get fairly complicated.

Also, this is the only cumulative variable that requires this extra treatment

Reply:
Oh good catch!